test/test_build.py

```diff
@@ -113,11 +113,20 @@ def build_images(shared_tmpdir, build_container, request, force_aws_upload):
     password = "password"
     kargs = "systemd.journald.forward_to_console=1"
 
+    default_ip = testutil.get_ip_from_default_route()
```

Comment:
Silly question - isn't `localhost` here good enough? And if not, maybe a short comment why not?

Reply:
No, as we need to copy the image from the host and from within the bib container